src/functTest/groovy/com/bmuschko/gradle/docker/tasks/container/DockerCreateContainerFunctionalTest.groovy

```diff
@@ -19,6 +19,7 @@ import com.bmuschko.gradle.docker.AbstractGroovyDslFunctionalTest
 import org.gradle.testkit.runner.BuildResult
 import org.gradle.testkit.runner.TaskOutcome
 
+
```

Comment:
Can you remove the extra new line?

Reply:
Sure!  Shoulda diffed it with master.  My IDE seems to be struggling with .editorconfig files, for whatever reason, at the moment, or I'da run a format on it.

Sidebar: I'm always both tempted and anti-tempted to go so far as to add Checkstyle, but it's not _too_ bad… :shrug: 

Thanks for getting at this so fast BTW!